Makefile

```
@@ -0,0 +1,71 @@
+##
+## ----------------
+## General
+## ----------------
+##
+
+COMPOSE_EXEC_PHP=docker-compose exec api
+
+all: help
+
+help: ## Display this message
+	@grep -E '(^[a-zA-Z0-9_\-\.]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+install: build start ## Bootstrap project
+	make api_composer CMD="install"
+
+start: ## Start
+	make -j 1 start-api
+
+start-api: ## Start API container
+	docker-compose up -d
+	docker-compose start
+	${COMPOSE_EXEC_PHP} symfony server:start -d
+
+stop: ## Stop containers
+	docker-compose stop
+
+ps: ## Display containers
+	docker-compose ps
+
+restart: stop start ## Restart containers
+
+build: ## Build csymfonyontainers
+	docker-compose build
+
+rm: ## Remove containers
+	make stop
+	docker-compose rm
+
+##
+## ----------------
+## Logs
+## ----------------
+##
+
+api_logs: ## Run API logs
+	${COMPOSE_EXEC_PHP} symfony server:log
+
+##
+## ----------------
+## Executable
+## ----------------
+##
+
+api_composer: ## Run composer commands
+	${COMPOSE_EXEC_PHP} symfony composer ${CMD}
+
+api_console: ## Run console command
+	${COMPOSE_EXEC_PHP} php bin/console ${CMD}
+
+##
+## ----------------
+## Quality
+## ----------------
+##
+
+api_lint: ## PHP linter
+	${COMPOSE_EXEC_PHP} ./vendor/bin/php-cs-fixer fix
```

J'ai une erreur :

```
$ make api_lint
docker-compose exec api ./vendor/bin/php-cs-fixer fix
Loaded config default.

In Finder.php line 611:
                                                                                 
  You must call one of in() or append() methods before iterating over a Finder.  
                                                                                 

fix [--path-mode PATH-MODE] [--allow-risky ALLOW-RISKY] [--config CONFIG] [--dry-run] [--rules RULES] [--using-cache USING-CACHE] [--cache-file CACHE-FILE] [--diff] [--format FORMAT] [--stop-on-violation] [--show-progress SHOW-PROGRESS] [--] [<path>...]

make: *** [Makefile:67: api_lint] Error 1
```

D'après une recherche web c'est parce qu'on n'a pas indiqué de chemin à fixer

Donc... ?

```suggestion
	${COMPOSE_EXEC_PHP} ./vendor/bin/php-cs-fixer fix src
```

---

J'ai corrigé, le fichier n'était pas commité :man_facepalming:

---

Il s'agit de ce fichier `api/.php-cs-fixer.dist.php`